docs/API/V1/README.md

```diff
@@ -35,12 +35,6 @@ artifacts, and the archived tarball. The resource ID becomes invalid subsequentl
 unless a new dataset is created with the same checksum value. (This is highly
 unlikely, unless the same Pbench Agent tarball is uploaded again.)
 
-### Users
-
-User resources are identified by a `username` property, which must be unique
-among all users registered on the Pbench Server. The user resource has a
-"user profile" that includes first and last name, and a contact email.
-
 A user resource is the "owner" of each [dataset](#datasets) managed by the
 Pbench Server. If a user is deleted, then any datasets owned by that user
 become orphaned; datasets with `public` access are still accessible to other
```

Comment:
With the "Users" header gone, users are no longer a Pbench Server "resource", so I think the sentence at (old) line 44 also needs to be reworked (and, we don't need a hyperlink here to the section that we're already in!  😀).  Since we're talking about datasets here, something like this might be good:
> Each dataset managed by the Pbench Server is initially "owned" by the user which uploaded it.

We no longer need to talk about user deletion, since that's external to the Pbench Server, and, if we describe access elsewhere, we don't need to mention it here.  Instead, you might mention that ownership of a dataset can be reassigned via the `ADMIN` user role (with the appropriate cross-reference).